jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryEndpointCaller.java

```diff
@@ -165,7 +165,7 @@ T call(RequestState requestState) throws IOException, RegistryException {
               .setAccept(registryEndpointProvider.getAccept())
               .setBody(registryEndpointProvider.getContent());
       // Only sends authorization if using HTTPS.
-      if (!isHttpProtocol) {
+      if (!isHttpProtocol || Boolean.getBoolean("sendCredentialsOverHttp")) {
```

Comment:
Oh, TIL system properties can be accessed through these kinds of methods. Interesting.